api/serializers/canteen.py

```diff
@@ -365,6 +365,62 @@ def get_central_kitchen_diagnostics(self, obj):
             return None
 
 
+class CanteenSummarySerializer(serializers.ModelSerializer):
+    images = MediaListSerializer(child=CanteenImageSerializer(), required=False)
+    diagnostics = FullDiagnosticSerializer(many=True, read_only=True, source="diagnostic_set")
+    central_kitchen_diagnostics = serializers.SerializerMethodField(read_only=True)
+
+    class Meta:
+        model = Canteen
+        my_fields = (
```

Comment:
```suggestion
        read_only_fields = (
            "id",
            "name",
            "city",
            "city_insee_code",
            "postal_code",
            "sectors",
            "daily_meal_count",
            "yearly_meal_count",
            "siret",
            "management_type",
            "production_type",
            "department",
            "region",
            "publication_status",
            "economic_model",
            "is_central_cuisine",
            "modification_date",
            # the following can still be improved
            "images",  # can return the first image only
            "diagnostics",
            "central_kitchen_diagnostics",  # can return a TD status instead of diagnostics
         )
```